docs/plugins/inputs/redis.asciidoc

```diff
@@ -59,42 +59,42 @@ input plugins.
 &nbsp;
 
 [id="plugins-{type}s-{plugin}-batch_count"]
-===== `batch_count`
+===== `batch_count` 
 
   * Value type is <<number,number>>
   * Default value is `125`
 
 The number of events to return from Redis using EVAL.
 
 [id="plugins-{type}s-{plugin}-data_type"]
-===== `data_type`
+===== `data_type` 
 
   * This is a required setting.
   * Value can be any of: `list`, `channel`, `pattern_channel`
   * There is no default value for this setting.
 
-Specify either list or channel.  If `data_type` is `list`, then we will BLPOP the
-key.  If `data_type` is `channel`, then we will SUBSCRIBE to the key.
-If `data_type` is `pattern_channel`, then we will PSUBSCRIBE to the key.
+Specify either list or channel.  If `redis\_type` is `list`, then we will BLPOP the
+key.  If `redis\_type` is `channel`, then we will SUBSCRIBE to the key.
+If `redis\_type` is `pattern_channel`, then we will PSUBSCRIBE to the key.
 
```

Comment:
REGRESSION:  Changed `redis\_type` to `data_type` in https://github.com/logstash-plugins/logstash-input-redis/pull/71.